environments/common/inventory/group_vars/all/defaults.yml

```diff
@@ -4,6 +4,8 @@ ansible_user: rocky
 appliances_repository_root: "{{ lookup('env', 'APPLIANCES_REPO_ROOT') }}"
 appliances_environment_root: "{{ lookup('env', 'APPLIANCES_ENVIRONMENT_ROOT') }}"
 appliances_environment_name: "{{ appliances_environment_root | basename | regex_replace('\\W+', '') }}" # [a-zA-Z0-9_] only
+appliances_protected_environments:
+  - prd
```

Comment:
As per [production cluster docs](https://github.com/stackhpc/ansible-slurm-appliance/blob/main/docs/production.md#environments-structure) we recommend environments `staging` and `production`, so this should be `production`.